Vendedor.cs

```diff
@@ -1,38 +1,57 @@
-namespace Exercicio_Salario
+namespace Exercicio_Salario;
+
+public class Vendedor
 {
-    public class Vendedor( string cpf, string cnpjEmpresa, string nome, decimal percentualComissao, decimal salarioFixo, List<Vendas> listaVendas)
+    public string Cpf { get; init; }
+    public string CnpjEmpresa { get; init; }
+    public string Nome { get; set; }
+    public decimal PercentualComissao { get; set; } 
+    public decimal SalarioFixo { get; set; } 
+    public List<Vendas> ListaVendas { get; set; }
+    public string InfosVendedor { get; set; }
+    public Empresa Empresa { get; set; }
+    
+    public Vendedor()
+    {
+        Cpf = "";
+        CnpjEmpresa = "";
+        Nome = "";
+        ListaVendas = [];
+    }
+        
+    public decimal CalcularSalario()
     {
-        public string Cpf { get; } = cpf;
-        public string CnpjEmpresa { get; } = cnpjEmpresa;
-        public string Nome { get; private set; } = nome;
-        public decimal PercentualComissao { get; set; } = percentualComissao;
-        public decimal SalarioFixo { get; set; } = salarioFixo;
-        public List<Vendas> ListaVendas { get; set; } = listaVendas; 
-        public Empresa Empresa { get; set; }
         
-        public decimal CalcularSalario()
-        {
-            if (ListaVendas == null || ListaVendas.Count == 0)
+            if (ListaVendas == null || ListaVendas.Count == 0 || PercentualComissao == 0)
             {
                 return SalarioFixo;
             }
-            
+            if (PercentualComissao < 0)
+            {
+                throw new Exception("Percentual de comissão não pode ser negativo");
```

Comment:
Não é uma boa práticar usar a classe exception, sempre que não tiver uma exceção que atenda sua necessidades o recomendado é você criar suas próprias exceções, facilitando o etendimento e suas futuras manutenções, segue documentação de apoio:

https://learn.microsoft.com/pt-br/dotnet/standard/exceptions/how-to-create-user-defined-exceptions